### ISSUE TITLE
DRUSH_BOOTSTRAP_DRUPAL_DATABASE fails on empty DB

### ISSUE THREAD

**Issue by clever-systems** [drush-ops/drush#608]
Doing somethin like this
```
  $items['populate-empty-db'] = array(
    'description' => dt('Populate empty DB.'),
    'bootstrap' => DRUSH_BOOTSTRAP_DRUPAL_DATABASE,
    'core' => array(7),
  );
```
We get errors like
```
Initialized Drupal 7.27 root directory at ...
Initialized Drupal site default at sites/default
<h1>Additional uncaught exception thrown while handling exception.</h1><h2>Original</h2><p>PDOException: SQLSTATE[42S02]: Base table or view not found: 1146 Table &amp;#039;mysite.blocked_ips&amp;#039; doesn&amp;#039;t exist: SELECT 1 FROM {blocked_ips} WHERE ip = :ip; Array
(
    [:ip] =&amp;gt; 127.0.0.1
)
 in drupal_is_denied() (line 1916 of /var/www/mysite/includes/bootstrap.inc).</p><h2>Additional</h2><p>PDOException: SQLSTATE[42S02]: Base table or view not found: 1146 Table &amp;#039;mysite.system&amp;#039; doesn&amp;#039;t exist: SELECT * FROM {system} WHERE type = &amp;#039;theme&amp;#039; OR (type = &amp;#039;module&amp;#039; AND status = 1) ORDER BY weight ASC, name ASC; Array
(
)
 in system_list() (line 165 of /var/www/mysite/includes/module.inc).</p><hr />Drush command terminated abnormally due to an unrecoverable error.
```

This is also reported here: [Cannot import to empty DB [#2053651] | Drupal.org](https://drupal.org/node/2053651)

**Comment by jonhattan** [comment]
Fixed in ff268e947949cc011b8a67472fad963908c05527

**Comment by clever-systems** [comment]
Awesome you fixed this, jonhattan.
